doc/sphinx/proof-engine/tactics.rst

```diff
@@ -1021,6 +1021,21 @@ Managing the local context
       the latter can still be referred to by a qualified name
       (see :ref:`Qualified-names`).
 
+   .. flag:: Avoid Local Imported Names
+
+      Name generation for names introduced by :tacn:`intro` and other
+      name introducing tactics avoids the names of objects (constants,
+      inductives, constructors) from the current open module and its
+      ancestors so that those objects may be referred to without
+      awkward quantification.
+
+      When this flag is on (it is off by default), imported names from
+      modules defined in the current file are also avoided.
+
+      .. deprecated:: 8.14
```

Comment:
Should be at least 8.15, right?  You plan to remove the flag soon, e.g. in 8.16?

Reply:
>Should be at least 8.15, right? 

yes

> You plan to remove the flag soon, e.g. in 8.16?

no